contracts/Strategy.sol

```diff
@@ -219,31 +292,55 @@ contract Strategy is BaseStrategy {
         if (_amountNeeded > wantBalance) {
             uint256 amountToFree = _amountNeeded.sub(wantBalance);
 
-            (, uint256 lastAccrued, ) = kashiPair.accrueInfo();
```

Comment:
why did you remove this?

Reply:
Accrue happens in `prepareReturn` and `liquidateKashiPair` so this is redundant.